test/ref/continuous_test.lst

```diff
@@ -162,9 +166,9 @@ Rician(1.0, 1.0)
 Rician(5.0, 1.0)
 Rician(10.0, 1.0)
 
-StudentizedRange(2.0, 2.0)
-StudentizedRange(5.0, 10.0)
-StudentizedRange(10.0, 5.0)
+#StudentizedRange(2.0, 2.0)
```

Comment:
Why is this commented out?

Reply:
This was suggested in https://github.com/JuliaStats/Distributions.jl/pull/1690#issuecomment-1458264073.

> You can work around them by commenting out the StudentizedRange references when regenerating the JSON file. And then only add the relevant Johnson-specific lines to this PR

I forgot to uncomment them back when I `git add` it. Will change them back!